apps/webapp/src/components/shared/toast-content.tsx

```diff
@@ -14,6 +14,20 @@ export const loadingTxToast: ToastFnProps = {
   duration: Infinity, // Building could potentially take a long time. Expected to be dismissed manually.
 };
 
+export const broadcastingTxToast = (): ToastFnProps => ({
```

Comment:
Given we aren't passing props, we can just make this an object and not a function